js/underscore-min.js

```diff
@@ -2,5 +2,7 @@
-//# sourceMappingURL=underscore-min.map
\ No newline at end of file
+// @license-end
+//# sourceMappingURL=underscore-min.map
```

Comment:
This file is minified on purpose, please remove the extra blank line at the end.  Other then that, good change! 👍